lib/src/csv_parser.cpp

```diff
@@ -103,8 +105,30 @@ Problem CsvParser::load_input() const {
     return problem;
 }
 
-void CsvParser::save_output(const Solution& sln) const {
-    UNUSED(sln)
+void CsvParser::save_output(const Problem& prb, const Solution& sln) const {
+	std::ofstream outfile("solution.csv");
+
+	outfile << "Route;" << "Vehicle;" << "Customer;" << "Arrive;" << "Begin;" << "End;" << "Leave;" << "Distance from previous;" << "Depot distance;" << std::endl;
+
+	char del = ';';
+
+	for (size_t i = 0; i < sln.routes.size(); ++i) {
+		for (size_t j = 0; j < sln.routes[i].second.size(); ++j) {
+
+			double prev_dist = 0;
+			if (j > 0) 
+				prev_dist = prb.costs[prb.customers[sln.routes[i].second[j]].id][prb.customers[sln.routes[i].second[j - 1]].id];
+			
+			outfile << i << del << prb.vehicles[sln.routes[i].first].id << del
+				<< prb.customers[sln.routes[i].second[j]].id << del <<
+				/*time 4 fields*/ prev_dist << del <<
```

Comment:
can stub time with zeroes or something. I want to be able to see the complete output after the use of this function